src/main/java/org/pf4j/update/UpdateManager.java

```diff
@@ -356,8 +368,12 @@ protected synchronized void initRepositoriesFromJson() {
         repositories = Arrays.asList(items);
     }
 
-    private Version getSystemVersion() {
+    private String getSystemVersion() {
         return pluginManager.getSystemVersion();
     }
 
+    private VersionManager getVersionManager() {
```

Comment:
For now, this method is not used by I wish to mark that it's possible to get a reference to VersionManager. In the end we can remove this method if it's not used.